util/repomirror/skopeomirror.py

```diff
@@ -124,14 +124,16 @@ def run_skopeo(self, args, proxy):
         stdout = ""
         stderr = ""
         while True:
-            stdout_nextline = job.stdout.readline()
-            stdout = stdout + stdout_nextline.decode("utf-8")
-            stderr_nextline = job.stderr.readline()
-            stderr = stderr + stderr_nextline.decode("utf-8")
+            stdout_nextline = job.stdout.readline().decode("utf-8")
+            stdout = stdout + stdout_nextline
+            stderr_nextline = job.stderr.readline().decode("utf-8")
+            stderr = stderr + stderr_nextline
             if stdout_nextline == "" and stderr_nextline == "" and job.poll() is not None:
                 break
-            logger.debug("Skopeo [STDERR]: %s" % stderr_nextline)
-            logger.debug("Skopeo [STDOUT]: %s" % stdout_nextline)
+            if stderr_nextline != "":
```

Comment:
It might be worth trimming all whitespace from `stderr_nextline` and `stdout_nextline` when being comparing with an empty string. At that point, you could also simply check its length. Example:
```
if not stdout_nextline.strip() and not stderr_nextline.strip() ...
```

Reply:
This is only on the python3 branch so customers won't see it. QE gets notes with each build but not in the form of our CHANGELOG.md.

Reply:
Ahh, I was under the assumption we were making notes about our Python3-specific changes as well. I've been adding them to my Pull Request descriptions in your new template field :) That's fine then